rollup.config.js

```diff
@@ -17,10 +17,7 @@ export default {
     buble({
       jsx: 'h',
       objectAssign: 'Object.assign',
-      transforms: {
-        asyncAwait: false,
-        forOf: false,
-      },
+      target: { chrome: 71, safari: 11.1, firefox: 64 }, // roughly disable ES feature transpilation
```

Comment:
do we need any transpiling? can we disable?

Reply:
this is the best way to disable transforms that i saw.

we need that jsx thing. if you know of another way to handle that i'm game.

disclaimer: first time i've messed with buble.